core/src/InputIcon/InputIcon.less

```diff
@@ -9,4 +9,7 @@
     font-size: @svg-icon;
     margin: 0;
   }
+  &.RoundedSearchIcon {
```

Comment:
Also drop the box shadow here

Is it possible to override from the search component LESS?